assets/sass/components/_icons.scss

```diff
@@ -0,0 +1,14 @@
+
+/* Bootstrap glyphicons */
+
+@mixin glyphicon {
```

Comment:
I was thinking the filename of this file. Currently, it only contains a mixing, which doesn't output any css as itself, hence it could be in a folder called `abstracts` or `utilities`. However, it defines glyphicon, which could be used as is and this component could someday extended to define a more icons, which would be used as is.

I think a good layout we could adapt is described in [this article](https://itnext.io/structuring-your-sass-projects-c8d41fa55ed4).

However, alternative would be using [@extend](https://sass-lang.com/documentation/at-rules/extend), which I didn't remember before. I'll add another comment about that in relevant location.

Reply:
Hmm.. That is an interesting point. I think our glyphicon indeed falls right in between the two categories "abstracts" and "components/icons". I myself prefer the name "icons", in the sense, that readily tells you, what to expect at this point. But that is beside the actual arguments, just my prefence.  

It really could be either way I think at this point of development.

**Within abstracts:**
- If were to have a lot of mixins, it would make sense to have it here. However, we don't yet have that much.
- It is only a mixin, should be in mixins. Like you said, doesn't output any CSS when compiled.

**Within components:**
- I think Matthew's original plan was to develop a library of re-usable components, so in that sense (future) icons fit well within the components-folder.
- Like you said, the glyphicon could be used as is. And I think this mixin is only (?) used for icons, wouldn't be used for anything else. Thus, it kinda makes sense to have it alongside the icons if more of them would be defined here. 
- If in the future there are other icons to be used or the change is made to fontawesome, this structure works with those cases.

However, this could always be changed, if we decide after working with Sass a bit more, that this is a poor place for it. So I think it works for now, and doesn't have to be set in stone :)

Reply:
I think we can keep the mixin here, if the `@extend` way is not better.